api/features/views.py

```diff
@@ -504,8 +506,6 @@ def get_queryset(self):  # type: ignore[no-untyped-def]
         """
         Override queryset to filter based on provided URL parameters.
         """
-        if getattr(self, "swagger_fake_view", False):
-            return FeatureState.objects.none()
```

Comment:
Why was this removed? I think this just moves the swagger generation warning from `get_serializer_context` to here instead.